### ISSUE TITLE
Is there a fork of this project?

### ISSUE THREAD

**Issue by Joe23232** [maidenone/ORGFXSoC#3]
I am very sure this project is dead. so I was curious Is there a fork of this project?

**Comment by maidenone** [comment]
not that i know of. but i would like to make a RISCV version. i have been looking into https://github.com/olofk/serv

i have a https://radiona.org/ulx3s/ to work with, but have not yet gotten started.

**Comment by Joe23232** [comment]
Ah I see. So would this be supporting normal desktops/towers? And would it be able to compete with Nvidia and AMD graphics cards? Are you the only person working on it or other people are working on it as well?

Also would it require any binary blobs like AMD and Nvidia require?

**Comment by maidenone** [comment]
no, take a look at this: https://en.m.wikipedia.org/wiki/RIVA_TNT

its way more capable then this project.

this project was developed by 2 students during 20 weeks. a nvidia card is developed by several hundred engineers for years.
no there are no binary blobs.

**Comment by Joe23232** [comment]
> its way more capable then this project.

Are you referring to this project https://github.com/olofk/serv?

**Comment by orkaboy** [comment]
Like @maidenone says, this was developed by two students during a limited period of time. It is very basic, and doesn't provide anything even close to contemporary graphics accelerators in terms of both functionality and performance.

**Comment by Joe23232** [comment]
@orkaboy Do you know any other projects that are being backed by a company (it can be small)?

**Comment by maidenone** [comment]
first off, what are you looking for?

**Comment by Joe23232** [comment]
@maidenone I just want a graphics card that is free and open source and is made for a desktop/tower, and it is binary blob free.

**Comment by maidenone** [comment]
i think the MIAOW project is the closest you can get.

I know there was some old PCI based project, but i cant find it. the
hardware cost was in the 1000$ range for very low performance. It was more
of a development kit and was not mass produced.

but take a look at the MIAOW, but yet again, its a research project, not a
product.


https://www.google.com/url?sa=t&source=web&rct=j&url=http://pages.cs.wisc.edu/~vinay/talks/HC27-MIAOW-Talk.pdf&ved=2ahUKEwjj1eSy7MfrAhVwwIsKHUD_CH0QFjADegQIAxAB&usg=AOvVaw3uwvzWbGVU3dIr4oLXWGW1

**Comment by Joe23232** [comment]
@maidenone Thanks mate :)
